media/robots.txt

```diff
@@ -32,13 +32,24 @@ Disallow: /*docs*$styles
 Disallow: /*docs*$toc
 Disallow: /*docs*$move
 Disallow: /*docs*$quick-review
+Disallow: /*docs*$samples
 Disallow: /*docs*$revert
 Disallow: /*docs*$repair_breadcrumbs
 Disallow: /*docs*$delete
 Disallow: /*docs*$restore
 Disallow: /*docs*$purge
 Disallow: /*docs*$subscribe
+Disallow: /*docs*$subscribe_to_tree
 Disallow: /*docs*$vote
 Disallow: /*docs.json
-Disallow: /*preview-wiki-content
+Disallow: /*docs/ckeditor_config.js
+Disallow: /*feed*
```

Comment:
This rule needs to be more precise as it will also block articles like https://developer.mozilla.org/en-US/Firefox/Releases/2/Adding_feed_readers_to_Firefox